packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -0,0 +1,249 @@
+
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+/* globals window */
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+import moment from 'moment';
+
+import peerConnectionManager from '../peer-connection-manager';
+import {ICESTATE, MEETINGS, RECHABILITY, _ANSWER_} from '../constants';
+
+import RechabilityRequest from './request';
+
+/**
+ * @class Rechability
+ */
+
+export default class Rechability extends StatelessSparkPlugin {
+  namespace = MEETINGS;
+
+  constructor(attrs, options) {
+    super({}, options);
+    this.rechabilityRequest = new RechabilityRequest({}, options);
+    this.orpheusCall();
+  }
+
+  /**
+   * fetches rechability data
+   * @returns {object} rechability data
+   */
+  orpheusCall = () => {
+    // Reset reachability result to empty to ensure no stale data
+    try {
```

Comment:
i still think just this try/catch block could be broken out as a function like
`attemptPersistedReachability()`

so that it could be used like such

`if (attemptPersistedReachability()) { return}`